index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>My portfolio</title>
+</head>
+<body>
+    <div class="toolbar-container">
+        <section class="toolbar">
+            <div class="time-container">
+                <p class="time">9:41</p>
+            </div>
+            <div class="status-container">
+                <img src="images/icon/Cellular Connection.png" alt="cellular connection icon" class="cellular-connection-icon">
+                <img src="images/icon/Wifi.png" alt="wifi icon" class="wifi-icon">
+                <img src="images/icon/Battery.png" alt="battery icon" class="battery-icon">
+            </div>
+        </section>
+        <nav class="nav-bar">
+            <div class="logo">
+                <p class="logo-placeholder">Feven Seyfu</p>
```

Comment:
Kindly fix your logo and use `<a>` element instead of `<p>`
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.